weave-js/src/components/PagePanelComponents/Home/Browse3/compare/ComparePageObjectsLoaded.tsx

```diff
@@ -0,0 +1,399 @@
+              <div className="flex items-center gap-4 px-16 pt-12">
+                {isSequential && (
+                  <Button
+                    size="small"
+                    disabled={first.version === 0}
+                    icon="back"
+                    variant="ghost"
+                    onClick={onPrev}
+                    tooltip="View previous versions"
+                  />
+                )}
+                <ShoppingCart
+                  items={cartItems}
+                  baselineEnabled={baselineEnabled}
+                  selected={selected}
+                />
+                {isSequential && (
+                  <Button
+                    size="small"
+                    disabled={
+                      lastVersionIndices &&
+                      last.version === lastVersionIndices[last.name]
+                    }
+                    icon="forward-next"
+                    variant="ghost"
+                    onClick={onNext}
+                    tooltip="View subsequent versions"
+                  />
+                )}
+              </div>
+              <div className="flex items-center gap-16 px-16 py-16">
+                <div className="flex h-28 items-center gap-4">
+                  <Switch.Root
+                    id="diffChangeSwitch"
+                    size="small"
+                    checked={onlyChanged}
+                    onCheckedChange={setOnlyChanged}>
+                    <Switch.Thumb size="small" checked={onlyChanged} />
+                  </Switch.Root>
+                  <label
+                    className="cursor-pointer select-none"
+                    htmlFor="diffChangeSwitch">
+                    Diff only
+                  </label>
+                </div>
+                {hasModes && (
+                  <div>
+                    <Button
+                      size="small"
+                      icon="split"
+                      variant="secondary"
+                      onClick={() => onSetMode('parallel')}
+                      active={checkedMode === 'parallel'}
+                      tooltip="Show double column view">
+                      Side-by-side
+                    </Button>
+                    <Button
+                      size="small"
+                      icon="content-full-width"
+                      variant="secondary"
+                      onClick={() => onSetMode('unified')}
+                      active={checkedMode === 'unified'}
+                      tooltip="Show single column view">
+                      Unified
+                    </Button>
+                  </div>
+                )}
+                <div className="flex-grow" />
+                <DropdownMenu.Root open={isOpen} onOpenChange={setIsOpen}>
+                  <DropdownMenu.Trigger>
+                    <div className="cursor-pointer">
+                      <div className="flex items-center gap-10">
+                        {baselineEnabled
```

Comment:
This toggle does not seem to do anything when there are only 2 things to compare

Reply:
I will hide it when there are only two things.